src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -33,48 +34,51 @@
 import picard.cmdline.CommandLineProgramProperties;
 import picard.cmdline.Option;
 import picard.cmdline.StandardOptionDefinitions;
-import picard.cmdline.programgroups.Alpha;
+import picard.cmdline.programgroups.Fingerprinting;
 
 import java.io.File;
-import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 /**
- * Attempts to check the sample identity of the sequence data in the provided SAM/BAM file
+ * Attempts to check the sample identity of the sequence/genotype data in the provided file (SAM/BAM or VCF)
  * against a set of known genotypes in the supplied genotype file (in either GELI or VCF format).
  *
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
         usage = CheckFingerprint.USAGE_DETAILS,
-        usageShort = "Computes a fingerprint from the supplied SAM/BAM file and compares it to the provided genotypes",
-        programGroup = Alpha.class // TODO -- when mature please move to a to-be-created Fingerprinting.class
+        usageShort = "Computes a fingerprint from the supplied input (SAM/BAM or VCF) file and compares it to the provided genotypes",
+        programGroup = Fingerprinting.class
 )
 public class CheckFingerprint extends CommandLineProgram {
 
-    static final String USAGE_DETAILS = "Computes a fingerprint from the supplied SAM/BAM file and " +
+    static final String USAGE_DETAILS = "Computes a fingerprint from the supplied input file (SAM/BAM or VCF) file and " +
             "compares it to the expected fingerprint genotypes provided. The key output is a LOD score " +
             "which represents the relative likelihood of the sequence data originating from the same " +
             "sample as the genotypes vs. from a random sample.  Two outputs are produced: (1) a summary " +
-            "metrics file that gives metrics related single read group (lane or index within a lane) " +
+            "metrics file that gives metrics at the single sample level (if the input was a VCF) or at the read " +
+            "level (lane or index within a lane) (if the input was a SAM/BAM) " +
             "versus a set of known genotypes for the expected sample, and (2) a detail metrics file that " +
             "contains an individual SNP/Haplotype comparison within a fingerprint comparison.  The two " +
             "files may be specified individually using the SUMMARY_OUTPUT and DETAIL_OUTPUT options.  " +
             "Alternatively the OUTPUT option may be used instead to give the base of the two output " +
             "files, with the summary metrics having a file extension '" + CheckFingerprint.FINGERPRINT_SUMMARY_FILE_SUFFIX + "' " +
             "and the detail metrics having a file extension '" + CheckFingerprint.FINGERPRINT_DETAIL_FILE_SUFFIX + "'.";
 
-    @Option(shortName=StandardOptionDefinitions.INPUT_SHORT_NAME, doc = "Input SAM or BAM file.")
+    @Option(shortName=StandardOptionDefinitions.INPUT_SHORT_NAME, doc = "Input file SAM/BAM or VCF.  (Single Sample)")
```

Comment:
I see that there are several places where a single EXPECTED_SAMPLE_ALIAS is assumed...still, in the case that a SAMPLE_ALIAS is provided, it could be used for both the  INPUT and the GENOTYPES files.